core/tests/widgets/canvas/test_canvas.py

```diff
@@ -159,6 +159,11 @@ def test_as_image(widget):
     assert image is not None
     assert_action_performed(widget, "get image data")
 
+    from PIL import Image as PIL_Image
+    image = widget.as_image(format=PIL_Image.Image)
+    assert type(image) == PIL_Image.Image
+    assert_action_performed(widget, "get image data as Pillow Image Object")
```

Comment:
The genreal structure of the test looks good; however, it should be broken out as a standalone test case. Each test case should be validating a single "thing that might break"; in this case, that thing is PIL support. It's entirely possible for PIL support to fail without `as_image()` to fail.